CMakeLists.txt

```diff
@@ -178,9 +178,7 @@ if(UNIX)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread")
 
     if(CMAKE_BUILD_TYPE STREQUAL "Coverage")
-       set(COVERALLS ON)
        include(${PROJECT_SOURCE_DIR}/cmake/Coverage.cmake)
-       turn_on_coverage()
```

Comment:
Are you sure this line should be removed?

Reply:
Yes because I also removed the macro inside the included file.

But a forgot to remove the old coveralls integration with appveyor.

Reply:
done, everything is OK.